test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -101,13 +154,18 @@
     public static final String DATA_CENTER = "datacenter1";
     public static final String DATA_CENTER_REMOTE = "datacenter2";
     public static final String RACK1 = "rack1";
+    private static final User SUPER_USER = new User("cassandra", "cassandra");
 
     private static org.apache.cassandra.transport.Server server;
     protected static final int nativePort;
     protected static final InetAddress nativeAddr;
     protected static final Set<InetAddressAndPort> remoteAddrs = new HashSet<>();
-    private static final Map<ProtocolVersion, Cluster> clusters = new HashMap<>();
-    protected static final Map<ProtocolVersion, Session> sessions = new HashMap<>();
+    private static final Map<Pair<User, ProtocolVersion>, Cluster> clusters = new HashMap<>();
+    protected static final Map<Pair<User, ProtocolVersion>, Session> sessions = new HashMap<>();
+
+    protected static final ThreadPoolExecutor schemaCleanup =
```

Comment:
I suspect that there'll be questions about WHY this executor's been added.
Better add some comment about the reasons for adding it.

Reply:
fixed in https://github.com/datastax/cassandra/pull/4

Reply:
fixed